crates/storage/src/params.rs

```diff
@@ -1,11 +1,11 @@
 use anyhow::Result;
 use pathfinder_common::{
-    BlockHash, BlockNumber, BlockTimestamp, ByteCodeOffset, CallParam, CallResultValue, CasmHash,
-    ClassCommitment, ClassCommitmentLeafHash, ClassHash, ConstructorParam, ContractAddress,
-    ContractAddressSalt, ContractNonce, ContractRoot, ContractStateHash, EntryPoint,
-    EventCommitment, EventData, EventKey, Fee, GasPrice, L1ToL2MessageNonce,
-    L1ToL2MessagePayloadElem, L2ToL1MessagePayloadElem, SequencerAddress, SierraHash,
-    StarknetVersion, StateCommitment, StorageAddress, StorageCommitment, StorageValue,
+    BlockCommitmentSignatureElem, BlockHash, BlockNumber, BlockTimestamp, ByteCodeOffset,
+    CallParam, CallResultValue, CasmHash, ClassCommitment, ClassCommitmentLeafHash, ClassHash,
+    ConstructorParam, ContractAddress, ContractAddressSalt, ContractNonce, ContractRoot,
+    ContractStateHash, EntryPoint, EventCommitment, EventData, EventKey, Fee, GasPrice,
+    L1ToL2MessageNonce, L1ToL2MessagePayloadElem, L2ToL1MessagePayloadElem, SequencerAddress,
+    SierraHash, StarknetVersion, StateCommitment, StorageAddress, StorageCommitment, StorageValue,
```

Comment:
Not this PRs problem, but I suggest replacing with `pathfinder_common::prelude::*` once it reaches this kind of mass..

Reply:
Ignore this comment, I think I'll create a refactoring issue to hunt down instances like this instead if we think this is a good idea.